Makefile

```diff
@@ -132,13 +132,13 @@ $(HELM):
 HCLOUD := $(abspath $(TOOLS_BIN_DIR)/hcloud)
 hcloud: $(HCLOUD) ## Build a local copy of hcloud
 $(HCLOUD):
-	curl -sSL https://github.com/hetznercloud/cli/releases/download/v1.42.0/hcloud-linux-amd64.tar.gz | tar xz -C $(TOOLS_BIN_DIR) hcloud
+	curl -sSL https://github.com/hetznercloud/cli/releases/download/v1.43.1/hcloud-linux-amd64.tar.gz | tar xz -C $(TOOLS_BIN_DIR) hcloud
 	chmod a+rx $(HCLOUD)
 
 KIND := $(abspath $(TOOLS_BIN_DIR)/kind)
 kind: $(KIND) ## Build a local copy of kind
 $(KIND):
-	go install sigs.k8s.io/kind@v0.22.0
+	go install sigs.k8s.io/kind@v0.23.0
```

Comment:
does CAPI bump kind as well? I thought they usually do that with every minor version

Reply:
@janiskemper this has nothing to do with go bump of CAPI. This is for installing latest version of kind (binary) in hack/tools/bin directory.

Reply:
what about this then? https://cluster-api.sigs.k8s.io/developer/providers/migrations/v1.6-to-v1.7

Reply:
CAPI updated it here https://github.com/kubernetes-sigs/cluster-api/pull/10610 

and we have an open PR for the same here https://github.com/syself/cluster-api-provider-hetzner/pull/1308

Reply:
they will launch it in 1.8. But since this is no go module, it might be less important to wait.